.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Feature request
+about: Suggest a new feature or a change.
+title: ''
+labels: 'enhancement'
+assignees: ''
+---
+
+<!-- Welcome! Thank you for contributing. These HTML comments will not render in the issue, but you can delete them once you've read them if you prefer! -->
+
+<!--
+Thanks for thinking of a way to improve Jupyter Scheduler. If this solves a problem for you, then it probably solves that problem for lots of people! So the whole community will benefit from this request.
+Before creating a new feature request please search the issues for relevant feature requests.
```

Comment:
Amend this to: "Before creating a new feature request please search the extant issues for relevant feature requests. See: https://github.com/jupyter-server/jupyter-scheduler/issues."